src/main/java/uk/gov/companieshouse/accountsDates/AccountsDates.java

```diff
@@ -65,25 +55,7 @@ public String convertLocalDateToDisplayDate(LocalDate date) {
      * @return
      * 
      */
-    public Map<String, String> getDateAndTime(String dateString) {
-        Map<String, String> timeObject = new HashMap<>();
-
-        Instant instant = Instant.parse(dateString);
-
-        LocalDateTime localDateTime = LocalDateTime.ofInstant(instant, ZoneId.of(ZoneOffset.UTC.getId()));
-        LocalDate localDate = localDateTime.toLocalDate();
-
-        DateTimeFormatter dateFormatDesired = DateTimeFormatter.ofPattern(DATE_FORMAT_D_MMMM_YYYY);
-        DateTimeFormatter timeFormatDesired = DateTimeFormatter.ofPattern("h:mm a");
-
-        String date = localDate.format(dateFormatDesired);
-        String time = localDateTime.format(timeFormatDesired).toLowerCase();
-
-        timeObject.put("date", date);
-        timeObject.put("time", time);
-
-        return timeObject;
-    }
+    public Map<String, String> getDateAndTime(String dateString);
```

Comment:
The JavaDoc is implementation specific details. A different implementation might have different time and date formats. Can you include "in the format 'D_MMMM_YYYY' 'h:mm a' " in the implementation JavaDoc instead please

Reply:
Resolved in 054c90f